crates/anymap/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "anymap"
+version = "0.0.0"
+description = "This crate is a port of only the necessary features from https://github.com/chris-morgan/anymap for use within rust-analyzer. Copyright © 2014–2022 Chris Morgan. COPYING: https://github.com/chris-morgan/anymap/blob/master/COPYING"
```

Comment:
We should probably mention the version here, especially given the license change.

Reply:
Just to confirm, as Veykril [commented](https://github.com/rust-lang/rust-analyzer/pull/15618#discussion_r1330541952), if we move the implementation of `anymap` to the `stdx` crate, we need to modify the Cargo.toml of `stdx` crate, and in the description, we need to mention the version of `anymap` we ported and the change of the license from `Blue Oak Model 1.0.0 or MIT or Apache-2.0` to `MIT OR Apache-2.0` as in rust-analyzer. Is my understanding correct?

Reply:
I changed my mind and I have added comments on a file in this commit. If it is better to move these comments to somowhere else, please let me know:)
[2b891ca](https://github.com/rust-lang/rust-analyzer/pull/15618/commits/2b891ca084f38aa88842cd8c568913adbf0e88a2)